pom.xml

```diff
@@ -253,6 +253,7 @@
     <module>sdk-actors</module>
     <module>sdk-springboot</module>
     <module>examples</module>
+    <module>dapr-spring-boot</module>
```

Comment:
Why are you creating all these new packages? We have purposely minimized our footprint on Spring boot. Before we can review this PR, we need a justification for the added support. We can hop into a Teams call for you to present the added benefits.

Reply:
Devs using Spring Boot are used to get Spring Boot starters to integrate other libs or frameworks. This was the basic idea. Just providing a starter for the DaprClient instance creation that can be used with autowiring afterwards in the source code. I've added a sample just showing this with the name 'dapr-spring-boot-samples-autostarter' in the samples folder of 'dapr-spring-boot-samples'. And this is fully optional. 

The Maven modules and resulting packages approach just follows the guidelines of the official Spring Boot documentation for the development of starters. Especially for samples I dont think your current approach to provide them all in one folder called 'examples', resulting in one jar package, is a good idea, because it completly screws up the dependencies and doesnt alllow for real application context setups, that are normally necessary in Spring based application. I definitely recommend to either put the examples in another repo or to follow the folder structure approach I've shown in 'dapr-spring-boot-samples', using one common parent pom but having an extra package/module/folder for each example.

I'm in the CEST time zone and ready to jump on a Teams call to discuss my proposed changes anytime, e.g. today 10 am Redmond time.